tsl/src/compression/compression_dml.c

```diff
@@ -745,7 +805,7 @@ decompress_chunk_walker(PlanState *ps, struct decompress_chunk_context *ctx)
  * indexes we ignore predicate.
  */
 static Bitmapset *
-compressed_insert_key_columns(Relation relation)
+compressed_insert_key_columns(Relation relation, bool *covering)
```

Comment:
What does "covering" mean here? In the index context it also means covering indexes, i.e. when they have a non-key payload columns.

Reply:
Added a comment,  the covering flag is set to true if we have a single constraint that is covered by all the columns present in the Bitmapset.

Reply:
i changed the name to covered